### DISCUSSION TITLE
Nuclear Dawn

### DISCUSSION THREAD

**Discussion by Battle8111** [seDirector/App#64] [Ideas]
Hello, 

It would be awesome to see Nuclear Dawn, dedicated server support, be added to the application. Though it be a dead and non-popular game, its still pretty fun and cheap for small communities to play on.

https://developer.valvesoftware.com/wiki/Dedicated_Servers_List

Nuclear Dawn Dedicated Server | (Steam CMD) 111710 |

**Comment by asheroto** [comment]
You should be able to use the `-game` option in server properties so long as it supports the traditional `srcds.exe` command line arguments that most games use. Since it's based on L4D I think it'll be fine.

seDirector will be supporting server templates in the coming months which will enable you to work with any game:
https://github.com/seDirector/App/issues/62
